test/integration-test-main.sh

```diff
@@ -798,17 +820,19 @@ function test_update_directory_time() {
     #
     # "touch -a" -> update ctime/atime, not update mtime
     #
-    touch -a $TEST_DIR
-    atime=`get_atime $TEST_DIR`
-    ctime=`get_ctime $TEST_DIR`
-    mtime=`get_mtime $TEST_DIR`
-    if [ $base_atime -eq $atime -o $base_ctime -eq $ctime -o $base_mtime -ne $mtime ]; then
-       echo "touch with -a option expected updated ctime: $base_ctime != $ctime, atime: $base_atime != $atime and same mtime: $base_mtime == $mtime"
-       return 1
+    if ! mount | grep s3fs | grep -e noatime -e relatime >/dev/null; then
```

Comment:
Same as above.

Reply:
Same as above. Please double check after I change the code.